integration_tests/mockApis/incentivesApi.ts

```diff
@@ -16,4 +16,208 @@ export default {
       },
     })
   },
+  stubGetAvailableLevels: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern: '/incentivesApi/iep/levels/MDI',
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: [
+          {
+            iepLevel: 'BAS',
+            iepDescription: 'Basic',
+            sequence: 1,
+            default: false,
+          },
+          {
+            iepLevel: 'STD',
+            iepDescription: 'Standard',
+            sequence: 2,
+            default: true,
+          },
+          {
+            iepLevel: 'ENH',
+            iepDescription: 'Enhanced',
+            sequence: 3,
+            default: false,
+          },
+        ],
+      },
+    })
+  },
+  stubGetReviews: (): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        urlPattern:
+          '/incentivesApi/incentives-reviews/prison/MDI/location/MDI-42/level/BAS\\?sort=NEXT_REVIEW_DATE&order=ASC&page=0&pageSize=20',
+      },
+      response: {
+        status: 200,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: {
+          locationDescription: 'Houseblock 1',
```

Comment:
Minor one - in the test we do `locationSelect().select('Houseblock 42')` but here we're returning a different houseblock number